test/preseed/provision_test.rb

```diff
@@ -21,10 +21,12 @@
     releases.each do |major, minor, release|
       describe "#{distro} #{major}.#{minor} #{release}" do
         it 'will compile' do
-          code, stdout, stderr = @renderer.render(distro, major, minor, release)
-          _(stdout).must_equal ''
-          _(stderr).must_equal ''
-          _(code).must_equal 0
+          skip("debconf-set-selections not found, use DEBCONFSETSEL to set path to executable") unless @renderer.debian_validator_present?
+          code, stdout, stderr, contents = @renderer.render(distro, major, minor, release)
+          puts contents if code != 0 || stdout != '' || stderr != ''
```

Comment:
Wouldn't it be better to let minitest do this? IIRC you can use `assert_equal(stdout, '', content)`.

Reply:
I actually had no idea, good advice.